README.md

```diff
@@ -40,12 +40,16 @@ Add a property of type string and configure the editor as seen in this example:
             reloadIfChanged: true
             inspector:
               group: 'text'
-              editor: 'Shel.Neos.ColorPicker/ColorPickerEditor'   
+              editor: 'Shel.Neos.ColorPicker/ColorPickerEditor'
+              editorOptions:
+                disablePicker: false        # optional
```

Comment:
As discussed in Slack I would also think we don't need the `disable` prefix and invert the option behaviour to have shorter options.

Reply:
Renamed the `editorOptions` to:

```
editorOptions:
  picker: bool
  fields: bool
  presetColors: array || bool
```